market/views.py

```diff
@@ -49,7 +71,12 @@ def create(request):
             return redirect(reverse('market:monitor', args=(new_market.market_id,)))
     elif request.method == 'GET':
         form = MarketForm()
-    return render(request, 'market/create.html', {'form': form})
+    
+    # code smell: path to should probably not be hard coded
```

Comment:
Should perhaps also have some error handling, if the file can't be read or parsed, defaulting to an empty list of default markets.

Reply:
I will change the format of the file containing the default market settings from .json to .py. Then we don't have to worry about reading or parsing the content

Reply:
Good idea